client/partition.go

```diff
@@ -89,7 +98,8 @@ func (c *PartitionClient) connect() (err error) {
 	if err != nil {
 		return fmt.Errorf("error connecting to partition leader: %w", err)
 	}
-	c.connLastUsed = time.Now().UTC()
+	c.connOpened = time.Now().UTC()
```

Comment:
What the reasoning behind using UTC location? AFAIS we are using it only with the combination of `time.Since()`. When `time.Since()` itself is a shortcut to  `time.Now().Sub(t)`, so I think it could lead to some confusion.

Reply:
just a convention i have, i always use `time.Now().UTC()`. it doesn't do anything here but that way i don't even have to think "does it matter or not". i just always set it to UTC.